tests/functional/TestKubeSmokeTest/run.sh

```diff
@@ -111,6 +111,13 @@ start_minikube() {
 		--memory=2048 \
 		--vm-driver=kvm \
 		--kubernetes-version="${KUBEVERSION}" || fail "Unable to start minikube"
+
+    # wait until it is ready
+    echo -e "\nWait until kubernetes containers are running and ready"
+    while [ 3 -ne $(kubectl get pods --all-namespaces | grep Running | wc -l) ] ; do
+        echo -n "."
+        sleep 1
+    done
```

Comment:
I'd say these should be two patches, since they are unrelated.
An "and" or "also" in the commit message is almost always an indicator that the patch should be split.